lib/client.js

```diff
@@ -338,16 +355,19 @@ class AssetComputeClient extends EventEmitter {
         });
     }
 
-
     async unregister() {
         if (!this.assetCompute) {
             await this.initialize();
         }
         const response = await this.assetCompute.unregister();
         this._registered = false; // wait until successful unregister to set registered variable
+
+        if (this.eventEmitter) {
+            await this.eventEmitter.close();
+            this.eventEmitter = undefined; // journal is removed, must stop eventEmitter
```

Comment:
Nit-pick: If it was used before, better reset it to null (undefined is more to mean it was never used before).